Examples/UseCases.ipynb

```diff
@@ -158,7 +182,11 @@
    "source": [
     "### Adding Annotations\n",
     "\n",
-    "Similar to [DKPro cassis](https://github.com/dkpro/dkpro-cassis), to add manual annotations to cas object, we need to defined it with `typesystem`. For the given text, annotations of Tokens that has an id and pos feature can be added in the following.\n"
+    "Similar to [DKPro cassis](https://github.com/dkpro/dkpro-cassis), to add manual annotations to cas object, we need to defined it with `typesystem`. For the given text, annotations of Tokens that has an id and pos feature can be added in the following.\n",
+    "\n",
+    "**REC**: The `CAS` type in Java UIMA is written with all upper-case letters. I think it would introduce an unnecessary difference to call it `Cas` here.\n",
```

Comment:
According to [pep8](https://www.python.org/dev/peps/pep-0008/#class-names) python format. Class names should normally use the CapWords convention. The uppercase convention is used for static variables. So, I am not sure, it would be a great idea to adopt.

Reply:
Java has the same convention. I guess the reason why CAS is all capitals in Java is because it is an abbreviation the long form being "Common Analysis System".

Reply:
With the JCas, the Java implementation also departs from the all capitals. So there are pro and con reasons - I don't have a particularly strong opinion - my suggestion to use `CAS` is because I think it may make it easier for people if the naming wouldn't arbitrarily differ between implementations. @jcklie

Reply:
I see your point, I'll update it.